docs/Breaking API Changes.md

```diff
@@ -71,3 +71,12 @@ All property setters now throw an exception.
 
 The `Workspace` and `DocumentId` parameters of `TextLoader.LoadTextAndVersionAsync(Workspace, DocumentId, CancellationToken)` are deprecated.
 The method now receives `null` `Workspace` and `DocumentId`.
+
+# Version 4.5.0
+
+`SymbolDisplayFormat.CSharpErrorMessageFormat` and `CSharpShortErrorMessageFormat` now include parameter names by default if used on a standalone `IParameterSymbol`.
+For example, parameter `p` in `void M(ref int p)` was previously formatted as `"ref int"` and now it is formatted as `"ref int p"`.
```

Comment:
FYI, we called these 'Display' apis intentionally so that we coudl change them without it being considered a breaking change.  I'm fine documenting this though, just wanted to reiterate our position that things that exist for 'display purposes' absolutely can and will change depending on what we think is best for displaying them :)